static/styles/criminals_map.css

```
@@ -0,0 +1,660 @@
+/* Add this new rule for dropdown options */
+#diplomat-selector option:not(:first-of-type) {
+    margin-top: 4px; /* Add space above each option except the first */
+}
+
+/* Limit dropdown height in Chrome/Safari */
+select#diplomat-selector {
+    background-color: var(--neutral-light);
+}
+
+/* This is a hack to limit dropdown height in modern browsers */
+@media screen and (-webkit-min-device-pixel-ratio: 0) {
```

Biome (2.1.2)

[error] Don't use unknown media feature names.

Unexpected unknown media feature name.
You should use media feature names defined in the CSS Specifications.

(lint/correctness/noUnknownMediaFeatureName)